src/news_gui.cpp

```diff
@@ -1000,7 +994,8 @@ void DeleteNews(Tpredicate predicate)
 void DeleteVehicleNews(VehicleID vid, AdviceType advice_type)
 {
 	DeleteNews([&](const auto &ni) {
-		return ((ni.reftype1 == NR_VEHICLE && ni.ref1 == vid) || (ni.reftype2 == NR_VEHICLE && ni.ref2 == vid)) && (advice_type == AdviceType::Invalid || ni.advice_type == advice_type);
+		/// FIX ME LOL NO
+		return ((ni.reftype1 == NR_VEHICLE && ni.ref1 == vid) || (ni.reftype2 == NR_VEHICLE && ni.ref2 == vid)) && (advice_type == AdviceType::Invalid);// || ni.advice_type == advice_type);
```

Comment:
Wouldn't the old code just work?
I know it previously used string IDs, so maybe an oversight when fixing this code conflicting?

Reply:
This one is undone now :)